community/prioritization_response.md

```diff
@@ -0,0 +1,58 @@
+## Overview
+
+Given that we are a small functional team that is meant to maintain :wrench:, support ::phone::, care for :heart:, and cultivate :sunflower: our amazing Octokit :octocat: and Terraform community we often have to make decisions that are best for the entire community.
+We don't take this act of prioritization lightly nor are we haphazard about it. We want to honor the time and thought that everyone has put into all of the SDKs that we have come to know and :heart:.
+
+Add that to the number of repos that we need to maintain (70+) and weekly interactions (500-1000 +/-) that we end up having with everyone I am sure it's clear why we have to define how we **prioritize** and our commitment to respond (think of an **SLA** of sorts) to those priorities.
+
+### What are the Priorities?
+
+In each of the repositories we own you'll see the same [labels](https://github.com/octokit/octokit.net/labels) for priority.  
+
+**They are:**  
+`Priority: Critical`, `Priority: High`, `Priority: Normal`, `Priority: Low`
+
+Incidentally, these map directly to the priorities set in our [Octokit Active community board](https://github.com/orgs/octokit/projects/10/views/4).
+
+| [Repo label](https://github.com/octokit/octokit.net/labels)   |      [Community board](https://github.com/orgs/octokit/projects/10/views/4) | Example |
+|----------|:-------------:|:-----------|
+| Priority: Critical | Urgent | Any security issues - Data leaks - Broken or non functioning SDK that has been published - Any stop work scenarios for users (where rollback is not possible)|
+| Priority: High | High | Any broken SDK methods - Deprecated or broken features - Critical dependency updates - Pull Requests that have been completed, are active, and are passing CI - Any issues that have been reported multiple times / impacting multiple users |
+| Priority: Normal | Normal | Any non-prioritized features - Issues impacting a low number of users - Schema changes and additions - Normal dependency updates |
+| Priority: Low | Low | Basic workflow changes - Draft issues/PRs - Stale issues/PRs - non-critical dependency updates - Any incomplete design work |
+
+### What are our response commitments?
+
+Note: This does not represent a commitment to resolution, just how quickly we'll respond and begin working on it
+
+| Priority  | Response expectation | 
+|----------|:-----------|
+| Urgent | 0 to **24hrs**:  Communication with users is ongoing and frequent until the issue is resolved. A security advisory will be posted if needed. |
```

Comment:
I'm a little nervous about committing to this. Does this mean an update at least every 24 hours on each urgent issue? Our current schedule right now doesn't budget for that, for example.

Reply:
Agreed, this was more a 👍 in the air that I was hoping to use for discussion.

Reply:
I like the idea definitely! I think my worry was introducing an expectation that we would provide daily support or updates until the issue is resolved, which I don't think is reflective of reality at this point unfortunately.